Mathlib/AlgebraicGeometry/EllipticCurve/Affine.lean

```diff
@@ -192,7 +193,7 @@ lemma equation_zero : W.Equation 0 0 ↔ W.a₆ = 0 := by
   rw [Equation, evalEval_polynomial_zero, neg_eq_zero]
 
 lemma equation_iff_variableChange (x y : R) :
-    W.Equation x y ↔ (W.variableChange ⟨1, x, 0, y⟩).toAffine.Equation 0 0 := by
+    W.Equation x y ↔ ((⟨1, x, 0, y⟩ : VariableChange R) • W).toAffine.Equation 0 0 := by
```

Comment:
This change causes extra writings here, since it can't recognize `⟨1, x, 0, y⟩ • W`. What about rename the old `variableChange` (which contains the actual definition) to `variableChange'` and make it private, and `abbrev variableChange : WeierstrassCurve R := C • W`?